src/blurredLocation.js

```diff
@@ -62,6 +61,8 @@ BlurredLocation = function BlurredLocation(options) {
     });
     var geometry = Blurred.responseJSON.results[0].geometry.location;
     options.map.setView([geometry.lat, geometry.lng],options.zoom);
+    $("#lat").val(geometry.lat);
```

Comment:
Oh wait. I see. Here we are doing both geocoding a string and moving the map. So this is correct, maybe we just need to give a better name to the function. geocodeStringAndPan()?

But maybe the val() code should go in a callback, like a function passed to geocodeStringAndPan (string, onComplete)... What do you think?